src/projects/suriano/config/englishSurianoLabels.ts

```diff
@@ -23,4 +23,15 @@ export const englishSurianoLabels = Object.assign({}, englishLabels, {
 
   // search results:
   results: "letters",
+
+  COM: "commission",
```

Comment:
Zijn deze 5 nodig bij Suriano?

Reply:
per wel, de rest weet ik niet, ik zal ze weghalen

Reply:
Fixed in [8988b27](https://github.com/knaw-huc/textannoviz/pull/154/commits/8988b2734d0fa14a4d9f8206dc51dc3b236b484f)